docs/sources/administration/api-keys/index.md

```diff
@@ -135,9 +135,36 @@ Complete the following steps to migrate from API keys to service accounts using
 1. Remove code that handles the old `/api/auth/keys` endpoint.
 1. Track the [API keys](http://localhost:3000/org/apikeys) in use and migrate them to SATs.
 
-### Migrate API keys to Grafana service accounts using Terraform
+#### Example
 
-This section shows you how to migrate API keys to Grafana service accounts using Terraform.
+Your current setup
+
+```sh
+curl -X POST -H "Content-Type: application/json" -d '{"name": "my-api-key", "role": "Viewer"}' http://admin:admin@localhost:3000/api/auth/keys
+
+# response from the api
+{"id":2,"name":"my-api-key","key":"eyJrIjoiTFRSN1RBOVc3SGhjblc0bWZodXZ3MnNDcU92Um5VZUIiLKJuIjoibXktYXBpLWtleSIsImlkIjoxfQ=="}%
+```
+
+New setup
+
+```sh
+# create a service account
+curl -X POST -H "Content-Type: application/json" -d '{"name": "my-service-account", "role": "Viewer"}' http://admin:admin@localhost:3000/api/serviceaccounts
+
+# response with the created service account id,name, login
+{"id":1,"name":"my-service-account","login":"sa-my-service-account","orgId":1,"isDisabled":false,"role":"Viewer","tokens":0,"avatarUrl":""}%
+
+# create the service account token with the service account id 1 - /serviceaccounts/{id} returned from the previous step
+curl -X POST -H "Content-Type: application/json" -d '{"name": "my-service-account-token"}' http://admin:admin@localhost:3000/api/serviceaccounts/1/tokens
+
+# response with the created SAT id,name and key
```

Comment:
Maybe worth adding something about how to use the key:
```suggestion
# response with the created SAT id, name and key. You can use the key the same way you used the API key.
```

Reply:
il make an example curl request with the newly created key as well